metadata/src/main/java/com/redhat/lightblue/metadata/parser/MetadataParser.java

```diff
@@ -328,6 +328,16 @@ public Index parseIndex(T object) {
                 }
                 parseProperties(index, object, INDEX_ELEMENTS);
 
+                // parse arbitrary index options, can be backend specific
+                T options = getMapProperty(object, "options");
```

Comment:
Similarly, you shouldn't need this., Any unrecognized property will be in 'properties'